src/mngt/ocf_mngt_cache.c

```diff
@@ -2077,6 +2077,13 @@ void ocf_mngt_cache_load(ocf_cache_t cache,
 	if (cache->metadata.is_volatile)
 		OCF_CMPL_RET(cache, priv, -EINVAL);
 
+	/* Load is not allowed with 'force' flag on */
+	if (cfg->force) {
+		ocf_cache_log(cache, log_err, "Starting cache with 'load' "
+				"and 'force' options simultaneously is forbidden.");
+		OCF_CMPL_RET(cache, priv, -EINVAL);
```

Comment:
`EINVAL` is not a valid OCF return code - but I kinda see why you put it here :) could you fix both?

Reply:
point for You